Analyses/src/BFieldPlotter_module.cc

```diff
@@ -44,6 +47,9 @@ namespace mu2e {
       fhicl::Atom<double>      axisTwoMin{Name("axisTwoMin"), Comment("Axis two lower edge (bin centered) for plotting (mm) (plane = x/y/z --> axis two = z/z/y)")};
       fhicl::Atom<double>      axisTwoMax{Name("axisTwoMax"), Comment("Axis two upper edge (bin centered) for plotting (mm) (plane = x/y/z --> axis two = z/z/y)")};
       fhicl::Atom<double>      mapBinSize{Name("mapBinSize"), Comment("Map bin size (mm) (must be a divisor of both axis lengths)"), 10.};
+      fhicl::Atom<bool>	       dump      {Name("dump"),       Comment("Dump map samples to a CSV file"), false};
+      fhicl::Atom<std::string> dumpName  {Name("dumpName"),   Comment("Suffix to the dump file name"),"Dump"};
+      fhicl::Atom<bool>	       detector  {Name("detector"),   Comment("Use Detector coordinate system"), false};
```

Comment:
The indentation in this block looks strange in github.  Are there tabs in the new code? If so, please replace with spaces.

Reply:
Good catch, there was a hidden tab.